Doc/requirements.txt

```diff
@@ -0,0 +1,4 @@
+# Requirements for docs build on netlify
+sphinx==2.0.1
```

Comment:
@Mariatta Do you think that this should be pinned or float upward?

Reply:
I pinned it because that's how it was in .travis.yml.

https://github.com/python/cpython/blob/master/.travis.yml#L58

Actually, I think it might be better to make .travis.yml to read from Doc/requirements.txt instead of specifying its own versions.

Reply:
Let's do this. Keep it pinned for now but add a comment: `# Pin sphinx to version specified in .travis.yml` on the line above Sphinx in requirements.txt

Reply:
Done.

Reply:
@epicfaace For now, the least disruptive change was to make the modification here. Ideally, there wouldn't be duplication. Would you like to open an issue suggesting the above.